cognite/client/data_classes/data_modeling/data_types.py

```diff
@@ -117,32 +119,37 @@ def _load(cls, resource: dict[str, Any], cognite_client: CogniteClient | None =
 
 
 @dataclass
-class Text(PropertyType):
+class ListablePropertyType(PropertyType, ABC):
+    is_list: bool = False
+
+
+@dataclass
+class Text(ListablePropertyType):
     _type = "text"
     collation: str = "ucs_basic"
 
 
 @dataclass
-class Primitive(PropertyType, ABC): ...
+class Primitive(ListablePropertyType, ABC): ...
```

Comment:
Seems to be unused, can we remove it? Or does it belong in the type hierarchy?
```suggestion
class Primitive(ListablePropertyType, ABC): ...
```

Reply:
We use it in the type hierarchy, and `pygen` and `neat` use it if memory serves.

Reply:
Are you sure?
```
>>> from cognite.client.data_classes.data_modeling.data_types import Primitive
>>> from tests.utils import all_subclasses
>>> all_subclasses(Primitive)
```

Reply:
Is is used `isinstance(propy.type, Primitive)`

Reply:
Either the search in my editor suddenly stopped working or I can't find that line in the SDK.